.circleci/config.yml

```diff
@@ -203,73 +203,86 @@ jobs:
           command: |
             cd samples/customer-registry-quickstart
             echo "Running mvn with SDK version: '$SDK_VERSION'"
+            # must also compile without the profile!
+            mvn -Dakkaserverless-sdk.version=$SDK_VERSION compile
```

Comment:
should it be `test-compile`?

Reply:
Ah, good idea, let's take it all the way there indeed.